pynucastro/networks/rate_collection.py

```diff
@@ -1989,7 +1989,7 @@ def _scale(arr, minval=None, maxval=None):
         scaled[scaled > 1.0] = 1.0
         return scaled
 
-    def gridplot(self, comp=None, color_field="X", rho=None, T=None, **kwargs):
+    def gridplot(self, comp=None, color_field="X", rho=None, T=None, minN=None, maxN=None, minZ=None, maxZ=None, **kwargs):
         """
```

Comment:
maybe we should use the same convention as in `plot()` to the parameters.  There we use `N_range` and `Z_range` and pass in a tuple with the range.  It would be good to be consistent.

Reply:
also note that this could have already been accomplished using a filter_function, like we already demonstrate in the docs:
https://pynucastro.github.io/pynucastro/plot-types.html

Reply:
The point here is that if I use a filter_function, the plot will resize its axes based on the maximum (N,Z) elements, which may result in several plots with different axes. The tuple indeed is the way to go